### ISSUE TITLE
[Announcement] Change in .NET Framework and .NET Core Issue filing 

### ISSUE THREAD

**Issue by kalaskarsanket** [microsoft/dotnet#1275]
This repository is no longer monitored or maintained for .NET Framework and .NET Core issues. We will be moving issues currently open in this repo to the appropriate repo listed below. If you have a new issue, please create it in the best repo from the list.  

- [Dev Community ](https://developercommunity.visualstudio.com/spaces/61/index.html)– for .NET Framework questions 
- [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/issues) - for ASP.NET Core questions 
- [dotnet/efcore ](https://github.com/dotnet/efcore/issues)- for Entity Framework Core questions 
- [dotnet/sdk -](https://github.com/dotnet/sdk/issues) for CLI tools and questions 
- [dotnet/runtime](https://github.com/dotnet/runtime/issues) - for runtime, API, or installer issues and questions 
- [dotnet/winforms](https://github.com/dotnet/winforms/issues) - for WinForms issues 
- [dotnet/wpf ](https://github.com/dotnet/wpf/issues)- for WPF issues 
- [nuget/home ](https://github.com/nuget/home/issues)- for NuGet questions and issues

**Comment by mairaw** [comment]
Shouldn't we officially archive and block new issues? /cc @terrajobst

**Comment by kalaskarsanket** [comment]
@mairaw  We are working on it, we were waiting on few confirmations as this repo also holds UWP release notes. The next step is to move the issues to appropriate locations and archive. 

@leecow @terrajobst

**Comment by MichaelSimons** [comment]
The https://github.com/dotnet/core/blob/master/Documentation/core-repos.md documentation directs to this repo (https://github.com/microsoft/dotnet) as _Catch-all repository for .NET (where most .NET Framework issues are filed)_.  Please update this reference accordingly before archiving/blocking new issues here.
